python/graphstorm/model/loss_func.py

```diff
@@ -110,9 +110,25 @@ class LinkPredictLossFunc(GSLayer):
 
```

Comment:
this class name and the docstring above doesn't make much sense. is this cross-entry loss? we refer to this as link prediction loss?

Reply:
A better name should be LinkPredictBinaryClassLossFunc